src/app/application.cpp

```diff
@@ -114,14 +114,14 @@ namespace
     const QString LOG_FOLDER = u"logs"_s;
     const QChar PARAMS_SEPARATOR = u'|';
 
-    const Path DEFAULT_PORTABLE_MODE_PROFILE_DIR {u"profile"_s};
+    const Path DEFAULT_PORTABLE_MODE_PROFILE_DIR{u"profile"_s};
```

Comment:
Seems you have applied some code formatting, which violates our coding style. You'll have to revert them. I'm not only referring to this line or file but this whole PR.